<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectInterface</class>
 <widget class="QDialog" name="SelectInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>520</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="NowInterface">
     <property name="text">
      <string>Now Interface:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>NowInterface</zorder>
  <zorder>comboBox</zorder>
  <zorder>tableView</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
